src/behaviours/__tests__/utils.tsx

```diff
@@ -33,6 +35,31 @@ export const drag = (element: any) => {
     return controls
 }
 
+export const asyncDrag = (element: any) => {
```

Comment:
this should become just a `drag` but that will require updating all the tests using `drag` at the moment, so I didn't want to include such changes as part of this PR - if you want I can work on migrating other tests to this one in a followup PR